src/site/content/en/blog/performance-tool-score-diffs/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Performance Tooling Score Differences
+subhead: Why do I get different scores in Test My Site, PageSpeed Insights, and Lighthouse?
+authors:
+  - egsweeny
+date: 2019-06-12
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+alt: Car speedometer.
+description: 
+  This post serves to explain the reasons that there are differences in performance scores between Test My Site (TMS), PageSpeed Insights (PSI), and Lighthouse. This document does not detail other differences in the tools, such as server run locations, network throttling presets, etc.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+[Load is not a single moment in time](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics) — it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. **This is why we generally use multiple metrics blended together to create our performance scores;** but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
+
+This post serves to explain the reasons that there are differences in performance scores between three Google tools: [Test My Site (TMS)](https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite), [PageSpeed Insights (PSI)](https://developers.google.com/speed/pagespeed/insights/), and [Lighthouse (LH)](https://developers.google.com/web/tools/lighthouse/). This document does **not** detail other differences in the tools, such as server run locations, network throttling presets, score variance between runs, etc.
+
+## Terminology
+
+Various terms about speed/performance are referenced in our tools. We want to start off with some clarification of how these terms are used.
+
+- **Page speed** refers to the speed of an individual page on your site (URL level).
+- **Site speed** refers to the speed of all of the pages on your site (origin level).
+
+It is important to note that "performance" or "speed" is not to be defined by a single metric (i.e. [First Contentful Paint (FCP)](/first-contentful-paint/), [Time to Interactive (TTI)](/interactive/), etc.). This is motivated by several considerations, including:
+- As mentioned in the introduction, 'page load is not a single moment in time—it's an experience that no one metric can fully capture' [source](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)
```

Comment:
Instead of linking short words like [here] or [source], it can improve the experience for folks using assistive technology to linkify the relevant text.
https://web.dev/labels-and-text-alternatives/#links

I went through and fixed other instances of this in the post.

Reply:
Thank you so much Rob... I really appreciate it.